frontend/resourcebundles/build.gradle

```diff
@@ -0,0 +1,3 @@
+jetbrainsAnnotations()
+
+addIntellijToCompileClasspath(withGit4Idea: false)
```

Comment:
Actually... going even further, is IntelliJ even needed on the classpath of `:frontend:resourcebundles` at all? 🤔